aptos-move/framework/aptos-experimental/sources/trading/market/market.move

```diff
@@ -237,6 +237,18 @@ module aptos_experimental::market {
         }
     }
 
+    public fun destroy_market<M: store + copy + drop>(self: Market<M>) {
```

Comment:
if this is to be used outside of tests, do we need to call callbacks to cleanup orders? or have two variants of destroy_market - with and without callbacks?

Reply:
tests need to destroy market with orders inside. 

we can additionally have destroy_empty() function , which aborts if non-empty (which inner structs already support)